### PR TITLE
SqrtX gate support in quri-parts-openqasm

### PR DESCRIPTION
**Pull request by snuffkin**
SqrtX(sx) is available in both QURI Parts and OpenQASM 3.0, so quri-parts-openqasm supports it.

stdgates.inc
https://github.com/openqasm/openqasm/blob/main/examples/stdgates.inc#L26

Sorry. I made a mistake and closed #50 , so I will pull request again.

### CODE REVIEW

File: packages/openqasm/quri_parts/openqasm/circuit/__init__.py [hunk 1]
```diff
@@ -43,9 +43,10 @@
     gate_names.Z: "z",
     gate_names.H: "h",
     gate_names.S: "s",
-    gate_names.Sdag: "sdg",
+    gate_names.SqrtX: "sx",
+    gate_names.Sdag: "sdag",
     gate_names.T: "t",
-    gate_names.Tdag: "tdg",
+    gate_names.Tdag: "tdag",
```
Comment (kwkbtr) [inline packages/openqasm/quri_parts/openqasm/circuit/__init__.py]: It seems that the changes in #49 are reverted, I guess it is unintentional.

Comment (snuffkin) [reply packages/openqasm/quri_parts/openqasm/circuit/__init__.py]: Sorry. I made a mistake.
I reflect the contents of #49.

**Review by kwkbtr — APPROVED** [review]
Thank you!

**Comment by kwkbtr** [comment]
Released [v0.9.2](https://github.com/QunaSys/quri-parts/releases/tag/v0.9.2) containing this update.

**Pull request merged**